staff/carlos-calvo/e-commerce-template/styles.css

```diff
@@ -0,0 +1,201 @@
+/*General*/
+*{
+    box-sizing: border-box;
+}
+/*Canvi de model de box, per a que sigui fins al border*/
+
+
+/*Seccion superior*/
+
+body{
+    background-image: url("https://i.pinimg.com/originals/4f/3f/fd/4f3ffdcbef67f88465abc5a953c91d8d.png");
+}
+
+main{
+    position: relative;
```

Comment:
instead of positioning it in relative way, you could also do it with the absolute position in reference to the body. Both  possibilities are ok if you know the different effect.